test/MyTested.AspNetCore.Mvc.Caching.Test/BuildersTests/DataTests/MemoryCacheTestBuilderTests.cs

```diff
@@ -115,7 +128,25 @@ public void ContainingEntryOfTypeShouldThrowExceptionWithIncorrectEntry()
         }
 
         [Fact]
-        public void ContainingEntryOfTypeAndKeyShouldNotThrowExceptionWithCorrectEntry()
+        public void ContainingEntryOfTypeShouldThrowExceptionWithIncorrectEntry()
+        {
+            Test.AssertException<DataProviderAssertionException>(
+                () =>
+                {
+                    MyController<MvcController>
+                        .Instance()
+                        .Calling(c => c.AddMemoryCacheAction())
+                        .ShouldHave()
+                        .MemoryCache(cache => cache.ContainingEntryOfType(typeof(int)))
```

Comment:
Put the `.ContainingEntryOfType` on a new line:

```
.MemoryCache(cache => cache
    .ContainingEntryOfType(typeof(int)).
```

Do the same everywhere you have such tests.